edx_event_bus_kafka/internal/consumer.py

```diff
@@ -594,6 +598,10 @@ def _add_message_monitoring(self, run_context, message, error=None):
                     # .. custom_attribute_description: The event type of the message. Note that the header in the logs
                     #   will use 'ce_type'.
                     set_custom_attribute('kafka_event_type', ",".join(event_types))
+            else:
+                # .. custom_attribute_name: kafka_received_message
+                # .. custom_attribute_description: True if we are processing a message with this span.
+                set_custom_attribute('kafka_received_message', False)
```

Comment:
Thanks. Could you update the couple of uses of `mock_set_custom_attribute` in the test: https://github.com/openedx/event-bus-kafka/blob/823645fe6fac91c7b61756b007dcac05df78af96/edx_event_bus_kafka/internal/tests/test_consumer.py#L233

Reply:
Added.